src/js/components/sidebar.jsx

```diff
@@ -100,22 +127,39 @@ const rotateUp = keyframes`
 `;
 
 const Wrapper = styled.section`
-  padding-top: 24px;
-  padding-left: 32px;
-  padding-right: 32px;
-  padding-bottom: 140px;
-  width: 25%;
-  max-width: 365px;
-  margin-top: 20px;
+
+  @media (max-width: 960px) {
+    width: auto;
+    margin-left: 40px;
+    margin-right: 40px;
+    padding-left: 72px;
+    max-height: ${(props) => (props.clicked ? '400px' : '230px')}
+  }
+  @media (max-width: 600px) {
+    max-height: 600px;
+    padding-right:32px;
+    padding-left: 20px;
+  }
+  @media (min-width: 960px) {
+    padding-left: 32px;
+    max-width: 365px;
+    padding-bottom: 140px;
+    width: 25%;
+    padding-top: 24px;
+    padding-left: 32px;
+    padding-right: 32px;
+    top: 0px;
+    bottom: 100px;
+    margin-top: 20px;
+  }
   margin-right: 40px;
-  border: 1px solid black;
   background: white;
   display: flex;
   flex-direction: column;
   flex: 1;
   position: -webkit-sticky;
   position: sticky;
-  top: 50px;
+  top: -1000px;
```

Comment:
I'm curious about this. Is this to ensure that the sidebar stays underneath the carousel? If so, how did you determined that -1000px was the right amount?